wandb/apis/public.py

```diff
@@ -424,7 +424,7 @@ def create_report(
         title: Optional[str] = "Untitled Report",
         description: Optional[str] = "",
         width: Optional[str] = "readable",
-        blocks: "Optional[wandb.apis.reports.util.Block]" = None,
+        blocks: Optional["Optional[wandb.apis.reports.util.Block]"] = None,
```

Comment:
what's going on here? should we just remove the `"Optional`?
```suggestion
        blocks: Optional["wandb.apis.reports.util.Block"] = None,
```

Reply:
yea, it's the tool I used to fix these, apparently it can't handle `Optional`'s. Fixed